content/apt/apt-1/contents.lr

```diff
@@ -43,7 +43,7 @@ Or nightly builds:
    deb-src https://deb.torproject.org/torproject.org tor-nightly-master-<DISTRIBUTION> main
    ```
 
-Replace `<DISTRIBUTION>` with your Operating System codename. Run `lsb_release -c` or `cat /etc/debian_version` to discover.
+Replace `<DISTRIBUTION>` with your Operating System codename. Run `lsb_release -c` or `cat /etc/debian_version` to check the Debian version you're running.
```

Comment:
Replace "check the Debian version" to "check the Operating System version", since it's possible to use our APT repository in Ubuntu and OS based on Debian.